docs/gym-skill.md

```diff
@@ -7,6 +7,14 @@ The AEA gym skill demonstrates how a custom Reinforcement Learning agent, that u
 
 Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
 
+##Discussion
+
+The `gym` example demonstrates the AEA framework's flexibility with respect to Reinforcement Learning using OpenAI's `gym` framework.
```

Comment:
The `gym` skills demonstrate how to wrap a Reinforcement Learning agent in a skill.